lib/mapper.js

```diff
@@ -52,6 +49,7 @@ exports.page = function(page, settings){
 
 exports.screen = function(screen, settings){
   var result = createCommonGAForm(screen, settings);
```

Comment:
@ndhoule It won't affect screen calls because if no url is passed as in screen calls, the hostname and path just won't get set as seen below.

![] (https://cloudup.com/c2q2KO2t_Cl+)

How did you add that code snippet above you code sorcerer?